### ISSUE TITLE
Two form controls in one FieldType Angular v14

### ISSUE THREAD

**Issue by James-Coates** [ngx-formly/ngx-formly#3455]
It seems the method used to include two form controls shown in issue #1222 no longer works for Angular v14.

The form values update but not the model.

Here is a stackblitz demonstrating the issue: [https://stackblitz.com/edit/angular-ceecss](https://stackblitz.com/edit/angular-ceecss)

Is there additional config now required to make this work or is there another way of achieving this?

**Comment by Greegko** [comment]
Alternative, you can turn your control into a fieldGroup, as how we solved this:
```
    {
      key: 'currency',
      fieldGroup: [
        { 
         key: 'name',
         type: 'input'
        },
        { 
          key: 'profession',
          type: 'select',
          props: {
            options: [...]
          }
         }
      ]
    }
```

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.1.7](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.1.7) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
